neo/SmartContract/ContractParameterType.cs

```diff
@@ -13,6 +13,7 @@ public enum ContractParameterType : byte
 
         Array = 0x10,
         Map = 0x12,
+        Null = 0x14,
```

Comment:
I think `Null` should not be a `ContractParameterType`. Because every type can be null.